notesapi/v1/tests/test_views.py

```diff
@@ -959,6 +959,26 @@ def test_pagination_page_start(self, page, start, previous_page, next_page):
             start=start
         )
 
+    @ddt.unpack
+    @ddt.data(
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد عمار", "tags": [u"عمار", u"Muhammad", u"محمد"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد", "tags": [u"محمد", u"Muhammad"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"Muhammad Ammar", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"محمد عمار", "search": u"Muhammad", "tags": [ u"Muhammad", u"عمار"]}
+    )
```

Comment:
I'm not clear on why all the different test cases and what they are testing, so it makes the test confusing to me.  For example, if we look at just the top 2 cases:
1. Is there anything special that happens with a search term broken by a space, or is this just a longer search term?
2. Does anything different happen when the tags are matched or not matched by the search string that we need to assert on?  Or, is the record just found if the search is matched in either the text or the tags?  If so, do you need more than 2 cases?  If you do, what is the english equivalent of describing each case?

Note: I think @jcdyer mentioned a clever way to use namedtuples (or something) to comment on ddt data.  If so, @jcdyer, can you send a link?